src/pyuvdata/uvbeam/uvbeam.py

```diff
@@ -2068,6 +2084,9 @@ def interp(
             Conversely, if the passed az/za are outside of the domain, they will be
             silently extrapolated and the behavior is not well-defined. Only
             applies for `az_za_simple` interpolation.
+        return_basis_vector : bool
+            Whether to return the interpolated basis vectors. Prior to v3.1.1 these
+            were always returned. Now they are not by default.
```

Comment:
This isn't right. The default is still False for this method, will become True in v3.3

Reply:
Ah, yes.

Reply:
So, I've left the default for now as True, and added the statement that in v3.3 it will be updated to be False.